### DISCUSSION TITLE
Only show Livestream in HLS Stream

### DISCUSSION THREAD

**Discussion by Gerbaum** [bluenviron/mediamtx#1351] [Questions]
Hi,
I have got a working HLS live stream, which i grab from an USB Camera with gstreamer.
I use 
`<iframe src="http://rtsp-simple-server-ip:8888/mystream" scrolling="no"></iframe>` 
in my webpage to show the stream.

Is it possible to just show the live stream without the possibility to jump back in the timeline? Also I noticed that if I change browser tab, the stream pauses und continues whenever i come back to that tab.
In my case browser should always show live stream whenever I come back to tab.

**Comment by aler9** [comment]
Hello,

> Is it possible to just show the live stream without the possibility to jump back in the timeline?

Yes, build a custom page starting from the [template](https://github.com/aler9/rtsp-simple-server/blob/main/internal/core/hls_index.html) and remove the `controls` attribute.

> Also I noticed that if I change browser tab, the stream pauses und continues whenever i come back to that tab.

This is not caused by the server but it's a browser policy. Videos without audio are automatically paused when not in focus.

[a comment by an automated account is not shown]
